undertone/undertone-htb-validator.js

```diff
@@ -39,8 +39,14 @@ function partnerValidator(configs) {
                 optional: true,
                 type: 'object',
                 properties: {
+                    ver: {
+                        type: 'string'
+                    },
+                    complete: {
+                        type: 'integer'
+                    },
                     nodes: {
-                        type: 'object'
+                        type: 'array'
```

Comment:
@tamarm-perion,
Thank you for the quick changes.

Can also update the nodes schema for the expected elements in the array? For example, nodes should be an array of SupplyChainNode objects.

Also, can we update the system tests to account for the new schain object being sent.